functional-test/pom.xml

```diff
@@ -839,6 +839,7 @@
                 <echo>-Dwebdriver.chrome.bin=full path to chrome binary.</echo>
                 <echo>-Dwebdriver.chrome.driver=full path to chromedriver binary. Default is chromedriver on $PATH</echo>
                 <echo>-Dwebdriver.wait=global wait time in seconds for element searches. Default is 10.</echo>
+                <echo>-Dwebdriver.screenshot.dir=location to store screenshots during test execution.Default is ${project.build.directory}/screenshots</echo>
```

Comment:
We should be, at least for the build server.

On 3 June 2015 at 14:51, Alex Eng <notifications@github.com> wrote:

> In functional-test/pom.xml
> <https://github.com/zanata/zanata-server/pull/820#discussion_r31593666>:
>
> > @@ -839,6 +839,7 @@
> >                  <echo>-Dwebdriver.chrome.bin=full path to chrome binary.</echo>
> >                  <echo>-Dwebdriver.chrome.driver=full path to chromedriver binary. Default is chromedriver on $PATH</echo>
> >                  <echo>-Dwebdriver.wait=global wait time in seconds for element searches. Default is 10.</echo>
> > +                <echo>-Dwebdriver.screenshot.dir=location to store screenshots during test execution.Default is ${project.build.directory}/screenshots</echo>
>
> I assume we are doing mvn clean on every built?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/820/files#r31593666>.
>